ui/imports/shared/popups/send/views/AmountToSend.qml

```diff
@@ -284,67 +288,88 @@ Control {
+                MouseArea {
+                    objectName: "amountToSend_mouseArea"
 
-                anchors.fill: parent
-                cursorShape: enabled ? Qt.PointingHandCursor : undefined
-                enabled: root.fiatInputInteractive
+                    anchors.fill: parent
+                    cursorShape: enabled ? Qt.PointingHandCursor : undefined
+                    enabled: root.fiatInputInteractive
 
-                onClicked: {
-                    const secondaryValue = d.secondaryValue
+                    onClicked: {
+                        const secondaryValue = d.secondaryValue
 
-                    d.fiatMode = !d.fiatMode
+                        d.fiatMode = !d.fiatMode
 
-                    if (textField.length === 0)
-                        return
+                        if (textField.length === 0)
+                            return
 
-                    const decimalPlaces = d.fiatMode ? root.fiatDecimalPlaces
-                                                     : root.multiplierIndex
-                    const divisor = SQUtils.AmountsArithmetic.fromExponent(
-                                      decimalPlaces)
+                        const decimalPlaces = d.fiatMode ? root.fiatDecimalPlaces
+                                                         : root.multiplierIndex
+                        const divisor = SQUtils.AmountsArithmetic.fromExponent(
+                                          decimalPlaces)
 
-                    const stringNumber = SQUtils.AmountsArithmetic.div(
-                        SQUtils.AmountsArithmetic.fromString(secondaryValue),
-                        divisor).toFixed(decimalPlaces)
+                        const stringNumber = SQUtils.AmountsArithmetic.div(
+                                               SQUtils.AmountsArithmetic.fromString(secondaryValue),
+                                               divisor).toFixed(decimalPlaces)
 
-                    const trimmed = d.fiatMode
-                                  ? stringNumber
-                                  : d.removeDecimalTrailingZeros(stringNumber)
+                        const trimmed = d.fiatMode
+                                      ? stringNumber
+                                      : d.removeDecimalTrailingZeros(stringNumber)
 
-                    textField.text = d.localize(trimmed)
+                        textField.text = d.localize(trimmed)
+                    }
                 }
+
+                HoverHandler { id: hoverHandler }
+
+                visible: !root.bottomTextLoading
+            }
+            StatusIcon {
+                Layout.preferredWidth: 16
+                Layout.preferredHeight: 16
+                Layout.alignment: Qt.AlignVCenter | Qt.AlignLeft
+
+                icon: "swap"
+                rotation: 90
+                color: Theme.palette.directColor5
+                visible: hoverHandler.hovered
+            }
+            RowLayout {}
```

Comment:
Why an empty layout here?

Reply:
Because without it, even if I assign alignment for max button to be `Qt.AlignRight` it sticks to the icon to the left.

Reply:
ah right, better use sth like a spacer then, e.g.:
```qml
Item { Layout.fillWidth: true }
```